_events/OSG-School-2024.md

```diff
@@ -0,0 +1,70 @@
+---
+short_title: OSG User School 2024
+title: "OSG User School 2024, Aug. 5–9"
+layout: events
+published: true
+excerpt: |
+    Applications for the OSG User School 2024 at the University of Wisconsin-Madison are now open!
+start_date: 2024-03-20
+end_date: 2024-08-09
+location: "University of Wisconsin-Madison"
+link: "https://osg-htc.org/school-2024/"
+image: "/assets/images/events/osg-user-school-2023-event.jpg"
+header_image: "/assets/images/events/osg-user-school-2023-event.jpg"
+---
+
+{% capture main %}
+
+**Could you tranform your research with vast amounts of computing?** 
```

Comment:
Miron has been disliking relative quantity words like “vast”. Maybe switch here to his preferred “Is limited computing capacity holding back your science?”